src/System.Threading.Tasks.Parallel/tests/Logger.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace CoreFXTestLibrary
+{
+    // A dummy class for passing through System.Console calls until the System.Console contract is available.
+    // To-do: Remove this workaround when possible.
+    internal static class Logger
+    {
+        internal static void LogInformation(string message, params object[] args)
+        {
+            System.Diagnostics.Debug.WriteLine(message, args);
```

Comment:
By default it won't write anything to console, so this won't clutter the screen when running tests.